### PR TITLE
Unify dataset dir for ldm bundles

### PR DESCRIPTION
**Pull request by yiheng-wang-nv**
Fixes # .

### Description
This PR does the following changes:

1. for `spleen_deepedit_annotation`, since the network it used is failed in monai==1.2.0rc6, it used 1.2.0rc5 instead. Now the network issue has been fixed in monai==1.2.0rc7, I changed the lib version in metadata.
2. for `models/brats_mri_axial_slices_generative_diffusion/configs/inference.json`, there is an issue that the create folder command is not executed, and run the inference command will raise an error. cc @Can-Zhao 
3. for other changes, the purpose is to unify the default value of `dataset_dir`.

### Status
**Ready**

### Please ensure all the checkboxes:

- [x] Codeformat tests passed locally by running `./runtests.sh --codeformat`.
- [ ] In-line docstrings updated.
- [ ] Update `version` and `changelog` in `metadata.json` if changing an existing bundle.
- [ ] Please ensure the naming rules in config files meet our requirements (please refer to: `CONTRIBUTING.md`).
- [ ] Ensure versions of packages such as `monai`, `pytorch` and `numpy` are correct in `metadata.json`.
- [ ] Descriptions should be consistent with the content, such as `eval_metrics` of the provided weights and TorchScript modules.
- [ ] Files larger than 25MB are excluded and replaced by providing download links in `large_file.yml`.
- [ ] Avoid using path that contains personal information within config files (such as use `/home/your_name/` for `"bundle_root"`).

### CODE REVIEW

File: models/brats_mri_axial_slices_generative_diffusion/configs/inference.json [hunk 1]
```diff
@@ -103,6 +103,7 @@
     "generated_image_np": "$@generated_image[0,0].cpu().numpy().transpose(1, 0)[::-1, ::-1]",
     "img_pil": "$Image.fromarray(visualize_2d_image(@generated_image_np), 'RGB')",
     "run": [
+        "$@create_output_dir",
```
Comment (yiheng-wang-nv) [inline models/brats_mri_axial_slices_generative_diffusion/configs/inference.json:106]: Hi @Can-Zhao , need your double confirm here, it seems `saver ` is not used. should it be removed?

Comment (yiheng-wang-nv) [reply models/brats_mri_axial_slices_generative_diffusion/configs/inference.json:106]: it's not urgent compared with the line I added, thus feel free to submit a new PR if need any changes.

**Review by Nic-Ma — APPROVED** [review]

**Review by mingxin-zheng — COMMENTED** [review]
Looks good to me Thx!

**Pull request merged**